libs/model-utils/src/lib/utils/many-to-one.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { manyToArray } from './many-to-array';
```

Comment:
this should test manyToOne util

Reply:
Damn, a copy/paste gone wrong, will put back the correct tests